earth_enterprise/src/installer/common.sh

```diff
@@ -153,6 +153,13 @@ show_no_tmp_dir_message()
     echo -e "scons release=1 installdir=$1 install\n"
 }
 
+show_corrupt_tmp_dir_message()
+{
+    echo -e "\nThe temp install directory specified [$1] is corrupt (failed on copy)."
+    echo -e "Please specify the path of the extracted install files or first run"
+    echo -e "scons release=1 installdir=$1 install\n"
```

Comment:
We should specify that installdir is optional, usually that's done using [installdir=$1]. 

We should probably also update this and the function above to use the stage_install target instead of install to match the instructions on the wiki.